independent-publisher-connectors/PaySpace/apiDefinition.swagger.json

```diff
@@ -0,0 +1,4476 @@
+            "description": "{{company-id}} The Company Id",
+            "x-ms-summary": "{{company-id}}"
+          },
+          {
+            "name": "CustomAuthHeader",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "description": "Bearer token value (without the keyword Bearer)",
+            "x-ms-summary": "Bearer token"
+          },
+          {
+            "name": "CustomEnvironmentHeader",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "description": "Variable that is either api (for PROD) or apistaging (for STAGING)",
+            "x-ms-summary": "Environment Variable"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "schema": {
+              "type": "string"
+            },
+            "required": true
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {}
+            }
+          }
+        }
+      }
+    },
+    "/odata/v1.1/{company-id}/EmployeePosition/effective/:{effectivedate}": {
+      "get": {
+        "summary": "Get a collection of positions as of an effective date",
+        "description": "Retrieves a list of employee position records as of the specified effective date and based on the specified company_id.",
+        "operationId": "GetACollectionOfPositionsAsOfAnEffectiveDate",
+        "parameters": [
+          {
+            "name": "$orderby",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "description": "(string) - Specifies the order in which items are returned",
+            "x-ms-summary": "{{$employee-field}}"
+          },
+          {
+            "name": "$top",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": " (integer($int32)) - Limits the number of items returned from a collection",
+            "x-ms-summary": "Optional $top"
+          },
+          {
+            "name": "$skip",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "description": " (integer($int32)) - Excludes the specified number of items of the queried collection from the result",
+            "x-ms-summary": "Optional $skip"
+          },
+          {
+            "name": "$count",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "description": " (true/false) - Return total count in response",
+            "x-ms-summary": "Optional $count"
+          },
+          {
+            "name": "$expand",
+            "default": "OrganizationGroups",
+            "in": "query",
+            "type": "string",
+            "required": true,
+            "x-ms-summary": "Optional $expand"
```

Comment:
@mint-virgil The 'description' property is required.

Reply:
updated in latest commit